src/lib/template-result.ts

```diff
@@ -20,6 +20,27 @@ import {reparentNodes} from './dom.js';
 import {TemplateProcessor} from './template-processor.js';
 import {boundAttributeSuffix, lastAttributeNameRegex, marker, nodeMarker} from './template.js';
 
+let policy: Pick<TrustedTypePolicy, 'createHTML'>|undefined;
+
+/**
+ * Turns the value to trusted HTML. If the application uses Trusted Types the
+ * value is transformed into TrustedHTML, which can be assigned to execution
+ * sink. If the application doesn't use Trusted Types, the return value is the
+ * same as the argument.
+ */
+function dangerouslyTurnToTrustedHTML(value: string): string|TrustedHTML {
```

Comment:
This function doesn't need to have such a scary name now that it's module-local.

Maybe something like`convertConstantTemplateStringToTrustedHTML` instead?